demo/index.html

```diff
@@ -20,7 +20,42 @@
       <h3>Basic mtz-file-download-behavior demo</h3>
       <demo-snippet>
         <template>
-          <mtz-file-download-behavior></mtz-file-download-behavior>
+
+          <file-downloader></file-downloader>
+
+          <dom-module id="file-downloader">
+            <template>
+              <button on-click="openUriInNewTab">Open In New Tab</button>
+              <button on-click="openUriInCurrentTab">Open In Current Tab</button>
+              <button on-click="createFile">Create File</button>
+            </template>
+            <script>
+              window.addEventListener('WebComponentsReady', () => {
+                Polymer({
+                  is: 'file-downloader',
+                  behaviors: [
+                    // Register use of the behavior
+                    mtz.FileDownloadBehavior
+                  ],
+                  // Open data at a URI in a new tab or the current tab allowing the user
+                  // to save if they so desire.
+                  openUriInNewTab() {
+                    this.downloadFromURI('https://github.com/MaritzSTL/mtz-file-download-behavior/tree/master/demo/sample.csv', true);
```

Comment:
I'm a bit confused. When you say a local reference, what are you referring to in this context?